contributed_definitions/nyaml/NXoptical_spectroscopy.yaml

```diff
@@ -240,7 +244,7 @@ NXoptical_spectroscopy(NXobject):
           exists: recommended
           doc: |
             Description of the detector type.
-          enumeration: [CCD, Photomultiplier, Photodiode, Avalanche-Photodiode, Streak Camera, Bolometer, Golay Detectors, Pyroelectric Detector, Deuterated Triglycine Sulphate, other]
+          enumeration: [CCD, photomultiplier, photodiode, avalanche-Photodiode, streak camera, bolometer, golay detectors, pyroelectric detector, deuterated triglycine sulphate, other]
```

Comment:
same like above and below

Reply:
resolved as underscores are not mandatory